Doc/library/typing.rst

```diff
@@ -1078,6 +1078,22 @@ The module defines the following classes, functions and decorators:
    ``Callable[..., Any]``, and in turn to
    :class:`collections.abc.Callable`.
 
+.. data:: Literal
+
+   A type that can be used to indicate to type checkers that the
+   corresponding value has a value literally equivalent to the
+   provided parameter.  For example::
```

Comment:
Writing "literally" here sounds strangely like the common (ab)use of "literally" in colloquial speech, e.g. "I was literally blown away by her performance".

More importantly, somehow this example doesn't convey why this would be an interesting thing to declare. I understand that you don't want to duplicate half the PEP in the docs, but maybe the example could still be a little more useful? Perhaps use a function and use a literal union in the example, and explain that the call must statically use one of the indicated literals. And then a link to the PEP.